docs/EULA.md

```diff
@@ -0,0 +1,147 @@
+# IBM Qiskit Code Assistant (Beta) Software End User License Agreement
```

Comment:
Is this file something we want in the `docs` dir or should it be at root? Currently we have no other docs in the docs dir, just images.

Reply:
if we leave it in docs should we move any other docs there too?

Reply:
Let's keep it in docs for now

Reply:
Only this file, to me it does not make sense to put it in the root dir since it not something documenting the repo per se

open to review to move other files to the same folder if needed in future PRs